Sources/Wrap.swift

```diff
@@ -340,11 +344,13 @@ extension NSArray: WrapCustomizable {
 }
 
 /// Extension customizing how NSDictionaries are wrapped
-extension NSDictionary: WrapCustomizable {
-    public func wrap(context: Any?, dateFormatter: DateFormatter?) -> Any? {
-        return try? Wrapper(context: context, dateFormatter: dateFormatter).wrap(dictionary: self as [NSObject : AnyObject])
+#if os(OSX) || os(iOS) || os(watchOS) || os(tvOS)
```

Comment:
The same argument can be made for if Apple introduces a new platform, then this code also needs to be updated. I don't think there's a way to make it 100% futureproof, so I think we should go for the simpler option for now 😄